bindata/bootkube/manifests/configmap-initial-kube-apiserver-client-ca.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: initial-client-ca
+  namespace: openshift-config
+data:
+  ca-bundle.crt: |
+    {{ .Assets | load "kube-ca.crt" | indent 4 }}
```

Comment:
this gives us as a starting point the one used by the installer to set everything up.